news/feat-mamba-dev-testing

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Adding libmamba/libmambapy as development dependencies within docker images (#191)
```

Comment:
```suggestion
* Adding libmamba/libmambapy as development dependencies within docker images (#191)
```

Move this to `Other`; this is not a user facing change, just internal dev stuff.

Reply:
Moved.